PokemonUnity.Shared/PokemonUnity.Shared.csproj

```diff
@@ -78,23 +78,9 @@
     <Compile Include="Extensions\MathHelper.cs" />
     <Compile Include="Extensions\ScriptConversion.cs" />
     <Compile Include="Extensions\StringHelper.cs" />
-    <Compile Include="Extensions\TransformExtension.cs" />
     <Compile Include="Game.cs" />
     <Compile Include="Game\Game.Database.cs" />
     <Compile Include="Game\Game.cs" />
-    <Compile Include="InBattle\Attack.cs" />
```

Comment:
everything in red, will and should eventually be transferred (moved) over to the frontend side of unity (or whatever platform user is building project on)